parmed/charmm/psf.py

```diff
@@ -458,6 +458,32 @@ def load_parameters(self, parmset):
         parmset : :class:`CharmmParameterSet`
             List of all parameters
 
+        copy_parameters : bool
+            if False, parmset will not be copied.
+
+            Warning:
+            -------
+            Not copying parmset will allow ParameterSet and Structure to share reference to types.
+            If you modify the original parameter set, the references in Structure list_types will be silently modified.
+            However, if you change any reference in the parameter set, then that reference will no longer be shared with
+            structure.
+
+            Example where the reference in ParameterSet is changed. This will NOT modify the paraemters in the psf.
+
+            psf.load_parameters(parmset, copy_parameters=False)
+            parmset.dihedral_type[('a1', 'a2', a3', a4)] = DihedralType(1, 2, 3)
```

Comment:
Use `angle_types` as an example instead of `DihedralType` -- this should actually be a `DihedralTypeList`, since `dihedral_types` is a Fourier sequence.